aiohttp/client_reqrep.py

```diff
@@ -812,7 +812,7 @@ def links(self) -> "MultiDictProxy[MultiDictProxy[Union[str, URL]]]":
 
                 link.add(key, value)
 
-            key = link.get("rel", url)  # type: ignore[assignment]
+            key = str(link.get("rel", url))
```

Comment:
MultiDict declares the keys to be `str`, so is this the right approach? `link["rel"]` is `Union[str, URL]`.